src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -267,9 +267,10 @@ def save(self) -> None:
 
     @rank_zero_only
     def finalize(self, status: str) -> None:
-        self.experiment.flush()
-        self.experiment.close()
-        self.save()
+        if status != "failed":
```

Comment:
Why don't we want to close it if it failed?

Reply:
@carmocca Hi, sorry for the late reply.
I've added the modification, but it makes `test_cli.py::test_cli_distributed_save_config_callback` fail.
I'd appreciate it if you help me with that.
FYI, I've found 10e133af6407463fe43951e9aa3153ff9b95c8c7 solves the problem as a workaround.